src/System.Management.Automation/engine/TypeTable_Types_Ps1Xml.cs

```diff
@@ -9198,6 +9198,53 @@ private void Process_Types_Ps1Xml(string filePath, ConcurrentBag<string> errors)
 
             #endregion System.Version#IncludeLabel
 
+#if UNIX
+            #region UnixStat
+
+
+            if (ExperimentalFeature.IsEnabled("PSUnixFileStat"))
```

Comment:
We could cache this, especially since it's an engine experimental feature, so it can't be toggled once PS has started

Reply:
Caching not required for perf reasons. I believe changes to all engine experimental features need a restart of PS.